code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -431,6 +431,19 @@
 		/obj/item/weapon/melee/sabre
 		)
 
+
+
+/obj/item/weapon/storage/belt/sabre/AltClick(mob/user)
+	if(contents.len)

```

Comment:
code\game\objects\items\weapons\storage\belt.dm:435:error: missing left-hand argument to &.